helix-core/src/main/java/org/apache/helix/controller/stages/ManagementModeStage.java

```diff
@@ -36,13 +38,30 @@
   @Override
   public void process(ClusterEvent event) throws Exception {
     // TODO: implement the stage
+    _eventId = event.getEventId();
     String clusterName = event.getClusterName();
+    ClusterStatusMonitor clusterStatusMonitor =
```

Comment:
NIT: This monitor will be management pipeline monitor, right? How about regular pipeline monitor update? Shall we update them as well?

Reply:
Please add some comments here. If we dont break the pipeline, following monitor update needs to be synchronized since the monitor is sharing by multiple pipelines.

Reply:
This monitor internally is synchronized. We don't have to explicitly synchronized it in the pipeline. The existing monitor update usage in default/task pipelines does not synchronized, either, which is fine.